db/migrate/20201022204510_add_extra_races_to_patient.rb

```diff
@@ -0,0 +1,9 @@
+class AddExtraRacesToPatient < ActiveRecord::Migration[6.0]
+  def change
+    add_column :patients, :unknown, :boolean
```

Comment:
I strongly agree, and I actually removed those fields in a later migration precisely because I hated those names. The migrations are a bit confusing because I didn't want to delete the old ones.